JS-Algorithms/challenges.js

```diff
@@ -71,6 +95,30 @@ The last 3 digits for the sum of powers from 1 to 10 is "317"
 
 const ownPower = (number, lastDigits) => {
   // YOUR CODE HERE...
+  let sumPow = 0;
+  let numPow = 1;
+
+  for(let evaluatedNum = 1; evaluatedNum <= number; evaluatedNum++){
+
+    numPow = 1;
+
+    for(let index = 0; index < evaluatedNum; index++){
+      numPow = numPow * evaluatedNum;
+    }
+
+    sumPow = sumPow + numPow;
+  }
+
+  sumPow = BigInt(sumPow);
+
+  let sumInString = sumPow.toString();
+  let finalString = "";
+
+  for(let finalDigit = lastDigits; finalDigit > 0 ; finalDigit--){
+    finalString = finalString + sumInString[sumInString.length - finalDigit];
+  }
```

Comment:
This works, just as a bonus challenge. try to solve this using `array.slice` instead